test/app-tap/gh-6330-table-new.test.lua

```diff
@@ -0,0 +1,16 @@
+#!/usr/bin/env tarantool
```

Comment:
Make your test executable. Otherwise it fails:

```
Traceback (most recent call last):                                             
  File "src/gevent/greenlet.py", line 906, in gevent._gevent_cgreenlet.Greenlet.run
  File "/home/sergeyb/sources/MRG/tarantool/test-run/lib/test.py", line 36, in _run
    self.callable(*self.callable_args, **self.callable_kwargs)        
  File "/home/sergeyb/sources/MRG/tarantool/test-run/lib/app_server.py", line 37, in run_server
    server.process = Popen(execs, stdout=PIPE, stderr=PIPE, cwd=cwd)
  File "/home/sergeyb/.local/lib/python3.8/site-packages/gevent/subprocess.py", line 808, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/home/sergeyb/.local/lib/python3.8/site-packages/gevent/subprocess.py", line 1793, in _execute_child
    raise child_exception                                                      
PermissionError: [Errno 13] Permission denied: '/home/sergeyb/sources/MRG/tarantool/test/app-tap/gh-6330-table-new.test.lua'
2021-08-16T15:25:03Z <TestRunGreenlet at 0x7f3826f6ad00 info='None'> failed with PermissionError    
                                       
[001] app-tap/gh-6330-table-new.test.lua                              [ fail ]
[001] Test failed! Output from reject file /home/sergeyb/sources/MRG/tarantool/build/test/var/rejects/app-tap/gh-6330-table-new.reject:
[001]                                                                          
[001] Last 15 lines of Tarantool Log file [Instance "app_server"][/home/sergeyb/sources/MRG/tarantool/build/test/var/001_app-tap/gh-6330-table-new.test.lua.ta
rantool.log]:
```

Reply:
I see this. Seems so scary
```
[027] app-tap/gh-6330-table-new.test.lua                              [ fail ]
[027] Test failed! Output from reject file /build/usr/src/debug/tarantool-2.9.0.298/test/var/rejects/app-tap/gh-6330-table-new.reject:
[027] 
[027] Last 15 lines of Tarantool Log file [Instance "app_server"][/build/usr/src/debug/tarantool-2.9.0.298/test/var/027_app-tap/gh-6330-table-new.test.lua.tarantool.log]:
[027]     Can't find log:
[027] Worker "027_app-tap" got failed test; restarted the server
[0
```

Reply:
Fixed